src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Automation/VsBuildManager.cs

```diff
@@ -97,19 +127,25 @@ void IEventSource<_dispBuildManagerEvents>.OnSinkRemoved(_dispBuildManagerEvents
         /// </summary>
         internal virtual void OnDesignTimeOutputDeleted(string outputMoniker)
         {
-            _threadingService.VerifyOnUIThread();
+            _threadingService.ExecuteSynchronously(async () =>
```

Comment:
This is always called on the UI thread, why did we change the assert?

Reply:
All of the `VerifyOnUIThread` was removed from this class in response to https://github.com/dotnet/project-system/pull/5326#discussion_r317933815 in the last PR.